crates/release_plz_core/src/release.rs

```diff
@@ -131,9 +136,24 @@ fn release_package(
         info!("published {}", package.name);
     }
 
+    if input.gh_release {
+        publish_release(package);
+    }
+
     Ok(())
 }
 
+async fn publish_release(package: &Package) {
+    let page = octocrab::Octocrab::default()
+        .repos(package.authors.join(", "), package.clone().name)
+        .releases()
+        .create(&package.version.to_string())
```

Comment:
This should be the tag name. It's different from the package version.
You can pass the `git_tag` variable above.